Update README.md (#4)
vtex-apps/disclosure-layout: docs/README.md

In your theme's `manifest.json` file, add the Disclosure Layout app as a dependency:

```diff
  "dependencies": {
+   "vtex.disclosure-layout": "1.x"
  }
```

Now, you are able to use all the blocks exported by the `disclosure-layout` app. Check out the full list below:

| Block name                   | Description                                                                                                                                                                                                                                                  |
| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| `disclosure-layout`          | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Parent block that enables you to build the disclosure layout using its 3 children blocks: `disclosure-trigger`, `disclosure-content`, and `disclosure-state-indicator`. |

> | `disclosure-trigger`         | Declares the blocks that will be rendered as disclosure indicators i.e. the blocks that when clicked on will trigger the disclosure content (defined by the `disclosure-content` block). |

Reviewer: ```suggestion
| `disclosure-trigger`         | Declares the blocks that will be rendered as disclosure triggers i.e. the blocks that when clicked on will open or close the disclosure content (defined by the `disclosure-content` block). |
```

> | `disclosure-content`         | Declares the blocks that are responsible for displaying the desired content when the disclosure indicator is clicked on. |

Reviewer: ```suggestion
| `disclosure-content`         | Declares the blocks that are responsible for displaying the desired content when the disclosure trigger is clicked on. |
```

| `initialVisibility` | `enum`                | Defines the initial visibility of the layout content. Possible values are: `visible` (content initially open) or `hidden` (content is only displayed with user interaction).    | `hidden`   |
| `animated`          | `boolean` | Whether or not the layout content should have animations. When set as `true`, this prop will enable additional `data-\*` attributes on the content which you can use as selectors in CSS. It will also ensure that the element will get hidden once the transition has ended. | `false` |

#### `disclosure-trigger` props

| Prop Name    | Type     | Description                                                                                          | Default value |
| ------------ | -------- | ---------------------------------------------------------------------------------------------------- | ------------- |
| `Show`       | `block`  | Name of the block that will be rendered when prompt to show the content. | `undefined`   |

> | `Hide`       | `block`  | Name of the block that will be rendred when prompt to hide the content.    | `undefined`   |

Reviewer: ```suggestion
| `Hide`       | `block`  | Name of the block that will be rendered when prompt to hide the content.    | `undefined`   |
```

| `as`         | `string` | HTML tag to be applied to the component when it is rendered on the UI. | `button`      |

> | `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `show` or `hide` props. | `undefined`   |

Reviewer: ```suggestion
| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `Show` or `Hide` props. | `undefined`   |
```

| `blockClass` | `string` | Block ID of your choosing to be used in [CSS customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property). | `undefined`   |

#### `disclosure-content` props

| Prop Name    | Type     | Description                                                                                          | Default value |
| ------------ | -------- | ---------------------------------------------------------------------------------------------------- | ------------- |
| `blockClass` | `string` | Block ID of your choosing to be used in [CSS customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization#using-the-blockclass-property). | `undefined`   |

> | `children`   | `block`  | Name of the block that will render the desired disclosure content. | `undefined`   |

Reviewer: ```suggestion
| `children`   | `block[]`  | List of blocks that will render the desired disclosure content. | `undefined`   |
```

| `maxVisible` | `enum` | Defines how many `disclosure-layout` blocks should be displayed at time. Possible values are: `one` (only one `disclosure-layout` block should have its content displayed at time) or `many` (different `disclosure-layout`'s contents can be displayed at time). | `one`  | 

#### `disclosure-trigger-group` props

| Prop Name    | Type     | Description                                                                                          | Default value |
| ------------ | -------- | ---------------------------------------------------------------------------------------------------- | ------------- |
| `Show`       | `block`  | Name of the block that will be rendered when prompt to show the content.            | `undefined`   |
| `Hide`       | `block`  | Name of the block that will be rendered when prompt to hide the content.            | `undefined`   |
| `as`         | `string` | HTML tag to be applied to the component when it is rendered on the UI.              | `button`      |

> | `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `show` or `hide` props.           | `undefined`   |

Reviewer: ```suggestion
| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `Show` or `Hide` props.           | `undefined`   |
```
